src/test/suite/mdbExtensionController.test.ts

```diff
@@ -1552,12 +1489,14 @@ suite('MDBExtensionController Test Suite', function () {
 
     let namespaceUsed = '';
 
+    const findStub = sinon.stub();
+    findStub.resolves([mockDocument]);
     const mockDataService = {
       find: (namespace: string) => {
         namespaceUsed = namespace;
         return Promise.resolve([mockDocument]);
       },
-    } as Pick<DataService, 'find'> as unknown as DataService;
+    } as unknown as DataService;
```

Comment:
Is there a reason why we're removing this type here? I'm thinking it might be unintended merge results. Looks like we have two find stubs here as well.

Reply:
The linter complaints with: `Promise-returning function provided to property where a void return was expected.eslint[@typescript-eslint/no-misused-promises](https://typescript-eslint.io/rules/no-misused-promises)`

Reply:
Is the lint complain inside of VSCode or in the `npm run check`? It might go away if you re-run `npm-install` and maybe restart vscode.
I'm thinking it's eslint keeping around the old `mongodb-data-service` interface which used callbacks instead of promises.